sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestFramework/DiagnosticVerifier.cs

```diff
@@ -34,6 +38,42 @@
 
 namespace SonarAnalyzer.UnitTest.TestFramework
 {
+
+    class ErrorInUnitTest : Exception
```

Comment:
A doc comment would be useful to make it obvious what this class is for - to add a dummy stack trace entry to the test result window to make it easy to navigate to the test resource file.

The class name is too generic; the class is intended to be used for errors found in test code in a resource file.